rust/catalyst-types/src/problem_report.rs

```diff
@@ -147,6 +159,18 @@ impl ProblemReport {
         !self.0.report.0.is_empty()
     }
 
+    /// Gets entries from the report.
+    #[must_use]
+    pub fn entries(&self) -> &ConcurrentVec<Entry> {
```

Comment:
I think here we could even return just an iterator
```suggestion
    pub fn entries(&self) ->  impl Interator<Item = Entry> {
```